build_image.sh

```diff
@@ -13,11 +13,12 @@
 }
 
 ## Retrieval short version of Git revision hash for partition metadata
-[ -z "$(git status --untracked-files=no -s --ignore-submodules)" ] || {
-    echo "Error: There is local changes not committed to git repo. Cannot get a revision hash for partition metadata."
-    exit 1
-}
-GIT_REVISION=$(git rev-parse --short HEAD)
+if [ -z "$(git status --untracked-files=no -s --ignore-submodules)" ]; then 
+    GIT_REVISION=$(git rev-parse --short HEAD)
+else
+    echo "Warning: There is local changes not committed to git repo. Cannot get a revision hash for partition metadata."
```

Comment:
For development it is not comfortably to keep in mind and after each change local commit should be performed. 
Another think is if build is run in second time sub-modules will have untracked files:
```
git status
...
        modified:   src/lldpd (modified content, untracked content)
        modified:   src/sonic-linux-kernel (untracked content)
        modified:   src/sonic-quagga (modified content, untracked content)
        modified:   src/sonic-sairedis (untracked content)
        modified:   src/sonic-swss (untracked content)
        modified:   src/sonic-swss-common (untracked content)
...
```
This also will cause "build_image.sh" fail.